tests/e2e/cypress/cypress/integration/acquisition/account-create_delete.spec.js

```diff
@@ -0,0 +1,67 @@
+/// <reference types="Cypress" />
+/*
+
+RERO ILS
+Copyright (C) 2020 RERO
+Copyright (C) 2020 UCLouvain
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, version 3 of the License.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+*/
+before(function () {
+  cy.fixture('account').then(function (accountData) {
+    this.account = accountData;
+  });
+  cy.fixture('users').then(function (userData) {
+    this.users = userData;
+  });
+  cy.fixture('common').then(function (commonData) {
+    this.common = commonData;
+  });
+})
+
+describe('Acquisition - Account', function() {
+  it('Create-Delete an account', function() {
+    const account = this.account.account;
+
+    cy.setup()
+    cy.adminLogin(this.users.librarians.virgile.email, this.common.uniquePwd);
+
+    // Go to Budgets Screen
+    cy.goToMenu('budgets-menu-frontpage');
+
+    // Click on the more recent year (top of the list)
+    cy.get(':nth-child(1) > ng-core-record-search-result > admin-budgets-brief-view.ng-star-inserted > .card-title > .pr-1').click()
+    cy.wait(1500)
+    // Click on "Add" button to create a new account
+    cy.get('.mt-4 > .btn').click()
```

Comment:
For me this selector isn't a good choice. You select something with a `btn` class into a parent with `margin-left-4`... how can you be sure than this is the add button ?